languages/elixir/exercises/concept/basics/.docs/introduction.md

```diff
@@ -1,44 +1,42 @@
-Elixir is a dynamically-typed language, meaning that the type of a variable is only checked at runtime. Using the match `=` operator, we can bind a value to a variable name:
+Elixir is a dynamically-typed language, meaning that the type of a variable is only checked at runtime. Using the match `=` operator, we can bind a value of any type to a variable name:
 
 ```elixir
-variable = 10 # Bind the integer value 10
-```
-
-Re-binding a variable's value is also done performed with the `=` operator. Once defined, a variable's type can change when re-bound.
+count = 1 # Bound an integer value of 1
+count = 2 # You may re-bind variables
 
-```elixir
-count = 1 # Bind an integer value of 1
-count = 2 # Re-bind to the new value of 2
-
-# Compiler does not error when binding a new type to the variable
-count = false
+count = false # You may re-bind any type to a variable
 ```
 
+## Modules
+
 Elixir is an [functional-programming language][functional-programming] and requires all named functions to be defined in a _module_. The `defmodule` keyword is used to define a module. All modules are available to all other modules at runtime and do not require an _access modifier_ to make them visible to other parts of the program. A _module_ is analogous to a _class_ in other programming languages.
 
-```elixir
-defmodule Calculator do
-  # ...
-end
-```
```

Comment:
I would keep this example here. It's new syntax, it makes it easier to visualize what the paragraph is talking about it, and it makes the later code example that uses `defmodule` a bit easier to process.